PlaceRouteHierFlow/router/GlobalGraph.cpp

```diff
@@ -265,6 +265,7 @@ void GlobalGraph::MST(int &WireLength, std::vector<pair<int, int> > &temp_path,
     std::vector<int> dest_set = temp_dest;
     SetSrcDest(src_set, dest_set);
     std::vector<int> temp_single_path = dijkstra(grid);
+    assert(!temp_single_path.empty());
```

Comment:
@soneryaldiz I added an assert here for empty path

Reply:
@854768750 , how is this assertion supposed to break the infinite loop which depends on `temp_dest`? It is not breaking in my local run.

Reply:
It will likely work in Debug mode, but not in Release mode because assertions are disabled.
You can throw an exception if you want it to fail in Release mode.

Reply:
Something like:
```
#include <stdexcept>

throw std::runtime_error("Empty path");
```

[runtime_error reference](https://en.cppreference.com/w/cpp/error/runtime_error)